src/utils/crypto.ts

```diff
@@ -143,7 +157,7 @@ export function generateKeyPair (raw = false) {
  * @param {Buffer} binaryData - Data to encrypt
  * @return {Uint8Array} Encrypted data
  */
-export function encryptKey (password, binaryData) {
+export function encryptKey (password: string, binaryData: string): Uint8Array {
```

Comment:
```suggestion
export function encryptKey (password: string, binaryData: Uint8Array): Uint8Array {
```
as I see from its usage in hd-wallet